CarpetXRegrid/README

```diff
@@ -0,0 +1,11 @@
+Cactus Code Thorn Regrid
+Author(s)    : Steven R. Brandt <sbrandt@cct.lsu.edu>
+Maintainer(s): Steven R. Brandt <sbrandt@cct.lsu.edu>
+Licence      : Cactus
```

Comment:
That's not a proper licence name

Reply:
True. @stevenrbrandt best might be to pick the same license that CarpetX uses.